src/logic/training_wheels.cc

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2020 by the Widelands Development Team
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ */
+
+#include "logic/training_wheels.h"
+
+#include <memory>
+
+#include "base/log.h"
+#include "io/filesystem/layered_filesystem.h"
+#include "logic/filesystem_constants.h"
+#include "scripting/lua_table.h"
+
+const std::string kTrainingWheelsFile = kSaveDir + g_fs->file_separator() + "training_wheels.conf";
```

Comment:
This doesn't belong in the `save` dir. Please create a new dir `~/.widelands/config` where this should go (and where the other config files should also be moved, but this could also be done later)

Reply:
And please move this to `filesystem_constants.h`

Reply:
> This doesn't belong in the `save` dir. Please create a new dir `~/.widelands/config` where this should go (and where the other config files should also be moved, but this could also be done later)

This is actually more difficult than it looks, because we also support the XDG standard. This will need a wider refactoring.

Reply:
I opened a new issue for this: https://github.com/widelands/widelands/issues/4383